src/assets/scss/main.scss

```
@@ -18,8 +18,22 @@ html, body {
   height: 100%;
 }
 
-h1, h2 {
-  font-weight: 900;
+h1 {
+  font-weight: 700;
+  font-size: 36px;
+}
+
+h2 {
+  font-weight: 700;
+  font-size: 26px;
+}
```

UI/UX Team specified letter spacing and color setup.
Should this be set or it's the same as the default value?

```suggestion
h1 {
  font-weight: 700;
  font-size: 36px;
  letter-spacing: 2px;
  color: #0E0E0E;
}

h2 {
  font-weight: 700;
  font-size: 26px;
  letter-spacing: 4px;
  color: #0E0E0E;
}
```

---

Originally, I was thinking about changing all of the colors/styles at once. But sure, thanks for the suggestion!